### DISCUSSION TITLE
oasst-data can't working with Python 3.8

### DISCUSSION THREAD

**Discussion by wannaphong** [LAION-AI/Open-Assistant#3528] [General]
Hello! I found Open Assistant Data Module (oasst_data) don't working with Python 3.8. I think oasst_data should can working with Python 3.8 because many deep learning library can't working with Python 3.10/3.11.

**Comment by x64x2** [comment]
> Hello! I found Open Assistant Data Module (oasst_data) don't working with Python 3.8. I think oasst_data should can working with Python 3.8 because many deep learning library can't working with Python 3.10/3.11.

Python 3.8 is outdated
